core-bundle/src/Resources/config/services.yml

```diff
@@ -628,3 +623,11 @@ services:
             - '@contao.routing.scope_matcher'
         tags:
             - { name: twig.extension }
+
+    contao.controller.frontend_module.two_factor:
+        class: Contao\CoreBundle\Controller\FrontendModule\TwoFactorController
```

Comment:
We could use the annotations now, can't we? 🤔

Reply:
jup, after rebasing

Reply:
But using annotations for everything would be a decision and should be applied to other stuff than 2FA